website/content/api-docs/secret/databases/index.mdx

```diff
@@ -273,12 +273,12 @@ This endpoint creates or updates a role definition.
   for this role.
 
 - `default_ttl` `(string/int: 0)` - Specifies the TTL for the leases
-  associated with this role. Accepts time suffixed strings ("1h") or an integer
+  associated with this role. Accepts time suffixed strings (`1h`) or an integer
   number of seconds. Defaults to system/engine default TTL time.
 
 - `max_ttl` `(string/int: 0)` - Specifies the maximum TTL for the leases
-  associated with this role. Accepts time suffixed strings ("1h") or an integer
-  number of seconds. Defaults to system/mount default TTL time; this value is allowed to be less than the mount max TTL (or, if not set, the system max TTL), but it is not allowed to be longer. See also [The TTL General Case](/docs/concepts/tokens#the-general-case).
+  associated with this role. Accepts time suffixed strings (`1h`) or an integer
+  number of seconds. Defaults to `sys/mounts`'s default TTL time; this value is allowed to be less than the mount max TTL (or, if not set, the system max TTL), but it is not allowed to be longer. See also [The TTL General Case](/docs/concepts/tokens#the-general-case).
```

Comment:
> Defaults to `sys/mounts`'s

I'm not sure about this one, but I don't see what "system/mount" would refer to otherwise?

(same question just above for `system/engine`: is this correct and if yes, what does it refer to?)

Reply:
yeah, I think you're correct on this one!